ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -284,6 +299,40 @@ const NetworksForm = ({
     isEditing,
   ]);
 
```

Comment:
nit: I see the 'new' prefix has been used to identify new components etc, but new is 'relative' especially to a dev arriving in the code and trying to merge a merge conflict etc, I would suggest a name like sortNetworksByOrderedList and would consdider annotating the old function being replaced as // TODO: with an explanation of why and when it should be removed, the // DEPRECATED: keyword might be usable in this case but I have not used it myself

Reply:
nit: should this function be wrapped in a useMemo so that if the orderedNetworkList doesnt change, then the orderedIndexMap is not recalculated on re-renders and clicks?